Content.Server/Administration/Rejuvenate.cs

```diff
@@ -1,51 +1,71 @@
-
-        public void Execute(IConsoleShell shell, IPlayerSession player, string[] args)
+        public string Description
         {
-            if (player == null)
+            get
             {
-                shell.SendText((IPlayerSession)null, "Player session is null.");
-                return;
+                var localizationManager = IoCManager.Resolve<ILocalizationManager>();
+                return localizationManager.GetString("Fully heals a mob.");
             }
-            if (args.Length < 1)
+        }
+        public string Help
+        {
+            get
             {
-                shell.SendText(player, Description);
-                return;
+                var localizationManager = IoCManager.Resolve<ILocalizationManager>();
+                return localizationManager.GetString("Usage: rejuvenate <mobUid_1> <mobUid_2> ... <mobUid_n>\nAttempts to heal the user's mob if no arguments are provided.");
             }
+        }
 
-            var entityManager = IoCManager.Resolve<IEntityManager>();
-            if (entityManager == null)
+        public void Execute(IConsoleShell shell, IPlayerSession player, string[] args)
+        {
+            var localizationManager = IoCManager.Resolve<ILocalizationManager>();
+            if (args.Length < 1 && player != null) //Try to heal the users mob if applicable
             {
-                shell.SendText(player, "Couldn't resolve IEntityManager.");
+                shell.SendText(player, localizationManager.GetString("Healing the user's mob since no arguments were provided."));
+                if (player.AttachedEntity == null)
+                {
+                    shell.SendText(player, localizationManager.GetString("There's no entity attached to the user."));
+                    return;
+                }
+                if (!player.AttachedEntity.TryGetComponent(out DamageableComponent damage))
+                {
+                    shell.SendText(player, localizationManager.GetString("The user's entity does not have a DamageableComponent."));
+                    return;
+                }
+                damage.HealAllDamage();
                 return;
             }
 
-            for (var i = 0; i < args.Length; i++)
+            var entityManager = IoCManager.Resolve<IEntityManager>();
+            foreach (var arg in args)
             {
-                var arg = args[i];
-                var entity = entityManager.GetEntity(EntityUid.Parse(arg));
+                if(!EntityUid.TryParse(arg, out var uid))
+                {
+                    shell.SendText(player, localizationManager.GetString("Could not find entity ") + arg);
+                    continue;
+                }
+                if(!entityManager.TryGetEntity(uid, out var entity))
+                {
+                    shell.SendText(player, localizationManager.GetString("Could not find entity ") + arg);
+                    continue;
+                }
```

Comment:
Just a heads up, these two statements could be combined into one `if(!EntityUid.TryParse(arg, out var uid) || !entityManager.TryGetEntity(uid, out var entity))` to reduce code duplication.

Reply:
Didn't consider doing that. Made the change 👍